docs/blog/posts/2024/containerssh-0.5.md

```diff
@@ -0,0 +1,172 @@
+9. [Removed the deprecated DockerRun and KubeRun backends](#removal-of-the-deprecated-dockerrun-and-kuberun-backends)
+
+## OAuth2 and Kerberos authentication
+
+The biggest change of this release is support for multiple authentication backends. Thanks to our contributors we now have support for OAuth2 and Kerberos authentication.
+
+OAuth2 authentication works with GitHub and any OIDC-compliant authentication server, such as Keycloak and Microsoft Active Directory Federation Services. We have actively worked with several SSH client vendors to make this authentication method work and we are happy to report that it works in OpenSSH, PuTTY, Filezilla, WinSCP, and more. The authenticaiton prompts the user to click on a link in their SSH client and then log in via their normal browser-based flow. What's more, you can automatically expose the GitHub or OIDC token to the container. Your users can directly use their credentials in your ContainerSSH environment.
+
+Similarly, Kerberos authentication is also useful in an enterprise setting. When users are logged in to their personal devices using company credentials, they will now be able to automatically log in to ContainerSSH with Kerberos. Optionally, users can also log in to ContainerSSH from a non-authenticated device using username and password, and ContainerSSH will automatically create a Kerberos ticket for them. This ticket is available in the container directly, so your users can work with their Kerberos credentials without any additional steps.
+
+[Read more »](/reference/auth){: .md-button}
+
+## Authorization webhook
+
+As part of our authentication and authorization overhaul we added a separate webhook. This webhook lets you match up the username entered in SSH and the authenticated credentials in a separate step. You can, for example, authenticate a user from Kerberos and then use a webhook to match up their Kerberos identity with the SSH username. 
+
+[Read more »](/reference/auth-webhook){: .md-button}
+
+## Metadata handling
+
+The authentication and configuration servers now support passing metadata between each-other and to ContainerSSH.
+
+[Read more »](/reference/auth-webhook#response){: .md-button}
+
+## Deploying files
+
+As part of the new metadata system both the authentication and configuration servers can now set environment variables and deploy files in the container user containers. This functionality depends on the ContainerSSH agent to be installed and available in the container image.
+
+[Read more »](/reference/auth-webhook#response){: .md-button}
+
+## Port, socket and X11 forwarding
+
+From this release support for forward and reverse port forwarding is supported natively. For these features the ContainerSSH agent has to be enabled as the agent acts as the entry & exit points of the connections inside the container.
+
+The features implemented correspond to the openssh commands:
+
+### Forward & Reverse port forwarding
```

Comment:
It would be worth mentioning that this will enable VSCode remote development with ContainerSSH.